db/data/autotest_files/py/specs.json

```diff
@@ -8,22 +8,32 @@
       },
       "test_data": [
         {
-          "script_files": ["test.py"],
-          "category": ["admin"],
+          "script_files": [
```

Comment:
Please revert the style changes in this file

Reply:
My mistake! Fixed this